static/src/javascripts/projects/commercial/modules/dfp/display-lazy-ads.ts

```diff
@@ -1,11 +1,33 @@
+import { isInVariantSynchronous } from 'common/modules/experiments/ab';
+import { commercialGptLazyLoad } from 'common/modules/experiments/tests/commercial-gpt-lazy-load';
 import { partition } from '../../../../lib/partition';
 import { dfpEnv } from './dfp-env';
 import { enableLazyLoad } from './lazy-load';
 import { loadAdvert } from './load-advert';
 
+// For users in variant group of commercialGptLazyLoad,
+// these ad slots will be be lazy loaded rather than loaded instantly.
+// TODO - remove if GPT lazy loading is fully rolled out.
 const instantLoadAdvertIds = ['dfp-ad--im'];
 
 const displayLazyAds = (): void => {
+	const useGptLazyLoad = isInVariantSynchronous(
+		commercialGptLazyLoad,
+		'variant',
+	);
+	if (useGptLazyLoad) {
+		window.googletag.pubads().enableLazyLoad({
+			// average mobile and desktop screen size ~ 1080px
```

Comment:
Pedantry :)

```suggestion
			// average mobile and desktop viewport height ~ 1080px
```

Reply:
We actually only have approximate screen size which we're using as a proxy for viewport height! 
I'll update the comment to explain that better.